bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/LimitsListener.java

```diff
@@ -135,6 +169,24 @@ public void propertyChange(PropertyChangeEvent evt) {
 						mm.addMarker(false, this, Severity.Warning, String.format("Observation reaches airmass %1.2f.", maxAirmass), v, a);
 					}
 				}
+
+				if (elevationIntervals.hasNext()) {
```

Comment:
These `hasNext` conditions should always be `True` but I left it just in case after getting so many null exceptions.

Reply:
> These hasNext conditions should always be True since if you are in a non NONE case, it means there is at least one elevation constraint.

It may be there there is a constraint but the night may not have any time at which the constraint is met?  If so the `elevationIntervals` would be empty.